src/pages/Error404Page/Error404Page.jsx

```diff
@@ -0,0 +1,26 @@
+import "./Error404Page.css";
+import { useNavigate } from "react-router-dom";
+import { useEffect,useState } from "react";
+export function Error404Page() {
+    const navigate = useNavigate();
+    const [counter, setCounter] = useState(10);
+    useEffect(() => {
+        counter > 0 && setTimeout(() =>setCounter(count=>count-1), 1000);
+        if(counter===0){
+            navigate("/")
+        }
+      }, [counter]);
```

Comment:
Suggestion: It is a good practice to clear the setTimeout before the component mounts again.

Reply:
thanks didn't knew that, I was thinking what should I return in useEffect, but thankyou  I got it